common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala

```diff
@@ -475,6 +475,14 @@ object JavaHelpers {
     in.toArray
   }
 
+  def collFrom(arr: Array[Long]): Coll[Long] = Colls.fromArray(arr)
+
+  def collFrom(arr: Array[Int]): Coll[Int] = Colls.fromArray(arr)
+
+  def collFrom(arr: Array[Boolean]): Coll[Boolean] = Colls.fromArray(arr)
+
+  def collFrom(arr: Array[Short]): Coll[Short] = Colls.fromArray(arr)
+
```

Comment:
Of course it is better to have Iso[Array[T], Coll[T]], but this is also good. :)

Reply:
The iso is not vital as from Java values of Coll can be accessed with coll.apply(i) - but it is not possible to construct a `Coll`